docs/Spark/pipeline-monitoring/enable-pipeline-monitoring.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Enable Pipeline Monitoring
+id: enable-pipeline-monitoring
+description: How to enable Pipeline Monitoring for Spark
+sidebar_position: 2
+tags:
+  - how-to
+  - setup
+  - monitoring
+  - spark
+---
+
+To use Pipeline Monitoring, you must enable it by updating the Prophecy Library version in your Project dependencies, and turn on the Pipeline Monitoring flag in Pipeline Settings.
+
+## Update Prophecy Library version
```

Comment:
This only mentions python projects. For scala projects, we need to mention scala prophecy libs version. 

Let's mention 8.0.23 as the minimum version.

Reply:
Done.